website/source/docs/job-specification/sidecar_task.html.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: "docs"
+page_title: "sidecar_service Stanza - Job Specification"
+sidebar_current: "docs-job-specification-sidecar-task"
+description: |-
+  The "sidecar_task" stanza allows specifying options for configuring
+  the task of the sidecar proxies used in Consul Connect integration
+---
+
+# `sidecar_task` Stanza
+
+<table class="table table-bordered table-striped">
+  <tr>
+    <th width="120">Placement</th>
+    <td>
+      <code>job -> group -> service -> connect -> **sidecar_task** </code>
+    </td>
+  </tr>
+</table>
+
+The `sidecar_task` stanza allows configuring various options for
+the sidecar proxy managed by Nomad for Consul Connect integration.
+It is valid only within the context of a connect stanza.
+
+```hcl
+ job "countdash" {
+   datacenters = ["dc1"]
+   group "api" {
+     network {
+       mode = "bridge"
+     }
+
+     service {
+       name = "count-api"
+       port = "9001"
+
+       connect {
+         sidecar_service {}
+         sidecar_task {
+            resources {
+               cpu = 500
+               memory = 1024
+            }
+         }
+       }
+     }
+     task "web" {
+         driver = "docker"
+         config {
+           image = "test/test:v1"
+         }
+     }
+   }
+ }
+
+```
+
+## `sidecar_task` Parameters
+- `name` `(string: )` - Name of the task.
+
+- `driver` `(string: )` - Driver used for the sidecar task.
+
+- `user` `(string:nil)` - Determines which user is used to run the task, defaults
+   to the same user the Nomad client is being run as.
+
+- `config` `(map:nil )` - Configuration provided to the driver for initialization.
+
+- `env` `(map:nil )` - Map of environment variables used by the driver.
+
+- `resources` <code>[resources][]</code> - Resources needed by this task.
+
+- `meta` `(map:nil )` - Arbitary metadata associated with this task that's opaque to Nomad.
```

Comment:
I don’t understand why but `Arbitary` was [caught by the `lint-go` job](https://circleci.com/gh/hashicorp/nomad/6780) on CircleCI?